src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/executor_pa.cpp

```diff
@@ -67,12 +67,11 @@ using namespace ov::intel_cpu;
 
 #    endif
 
-template <
-    typename T,
-    ov::element::Type_t SRC_PREC,
-    std::enable_if_t<(std::is_same_v<T, ov::bfloat16> || std::is_same_v<T, ov::float16> || std::is_same_v<T, float>) &&
-                         (SRC_PREC != ov::element::u8 || SRC_PREC != ov::element::u4),
```

Comment:
@zhangYiIntel I believe the intention here was to write (!= u8 && !=u4), right? Otherwise, this check is simply always true.

Reply:
Hi Egor you are right, in my PR, I just remove them since this condition is no longer used.
https://github.com/openvinotoolkit/openvino/pull/31054#discussion_r2227862457